lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -595,10 +604,16 @@ handle_op(Rest, Line, Column, Kind, Length, Op, Scope, Tokens) ->
 
 handle_comments(CommentTokens, Tokens, Scope) ->
   case Scope#elixir_tokenizer.preserve_comments of
-    true  -> lists:append(CommentTokens, Tokens);
```

Comment:
Since we're not touching this line, let's skip this change as it adds noise to the PR and entries to the blame for just style changes.